### PR TITLE
Update Java client to use unary requests

### PR DESCRIPTION
**Pull request by jul-sh**
First time writing Java! Was fun, shoutout to @ipetr0v for teaching me how to log objects the Java Way [TM] in debugging
```
    byte[] bytes = streamingResponse.getBody().toByteArray();

    byte[] HEX_ARRAY = "0123456789ABCDEF".getBytes(StandardCharsets.US_ASCII);
    byte[] hexChars = new byte[bytes.length * 2];
    for (int j = 0; j < bytes.length; j++) {
        int v = bytes[j] & 0xFF;
        hexChars[j * 2] = HEX_ARRAY[v >>> 4];
        hexChars[j * 2 + 1] = HEX_ARRAY[v & 0x0F];
    }
    String s = new String(hexChars, StandardCharsets.UTF_8);

    logger.log(s);
```

### CODE REVIEW

File: oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java [hunk 1]
```diff
@@ -123,46 +123,25 @@ public void attest(ManagedChannel channel, Predicate<ConfigurationInfo> verifier
...
-      @Override
-      public void onError(Throwable t) {
-        Status status = Status.fromThrowable(t);
-        logger.log(Level.WARNING, "Couldn't receive response: " + status);
-      }
-
-      @Override
-      public void onCompleted() {}
-    };
-    requestObserver = stub.stream(responseObserver);
```
Comment (jul-sh) [inline oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java:152]: This is pretty nice, using unary requests allows us to use the blocking gRPC api for calls. This means we no longer have to implement our own blocking queue to wait for server responses

File: oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java [hunk 2]
```diff
@@ -47,16 +48,15 @@
...
+import oak.session.unary.v1.UnaryRequest;
+import oak.session.unary.v1.UnaryResponse;
+import oak.session.unary.v1.UnarySessionGrpc;
 
 /**
  * Client with remote attestation support for sending requests to an Oak Functions application.
  */
 public class AttestationClient {
-  static public final Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(5);
+  public static final Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(5);
```
Comment (jul-sh) [inline oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java:59]: unrelated to this PR, but fixed for consistency

File: oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java [hunk 3]
```diff
@@ -123,46 +123,25 @@ public void attest(ManagedChannel channel, Predicate<ConfigurationInfo> verifier
...
-        Status status = Status.fromThrowable(t);
-        logger.log(Level.WARNING, "Couldn't receive response: " + status);
-      }
-
-      @Override
-      public void onCompleted() {}
-    };
-    requestObserver = stub.stream(responseObserver);
+    SecureRandom random = new SecureRandom();
+    byte[] rawSessionId = new byte[8];
```
Comment (ipetr0v) [inline oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java:129]: nit: Could you add this as a constant?

Comment (jul-sh) [reply oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java]: thx done

**Review by conradgrobler — APPROVED** [review]

**Pull request merged**